doc(release): Prepare release notes for collect 23.10 (#2956)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

## Centreon Collect

### 23.10.2

Release date: `January 11, 2024`

#### Centreon Broker

<details>
  <summary>Bug fixes</summary>

> - [Broker] Fixed a bug concerning the update of a Business Activity, after status change of a KPI with type BA.

Reviewer: ```suggestion
- [Broker] Fixed a bug occurring when a Business Activity was updated after a status change for a KPI with a BA type.
```

- [Install] Fixed rights on SSH directory for Debian.

> - [Packaging] Fixed an error that was returned during centreon-broker-cbmod installation.

Reviewer: ```suggestion
- [Packaging] Fixed an error that was returned during the installation of **centreon-broker-cbmod**.
```
